CMakeLists.txt

```diff
@@ -49,6 +53,16 @@ option(ENABLE_SANITIZE "Sanitize tool" OFF)
 option(ENABLE_SECURITY "Security Ford protocol protection" ON)
 option(ENABLE_HMI_PTU_DECRYPTION "Policy table update parsed by hmi" ON)
 option(ENABLE_EXTENDED_POLICY "Turns extended flow which requires embedded system interaction" ON)
+option(USE_COTIRE "Use Cotire to speed up build (currently only for commands tests)" ON)
```

Comment:
@LuxoftAKutsan I attempted the build again with 8GB of RAM allocated to my VM, and the same error continued to appear.

I did find that if I built with `USE_COTIRE=OFF` first and then build again with `USE_COTIRE=ON` without clearing the build directory, the build succeeds (after using ~2.5GB of RAM to build the commands tests). 

This seems to me like it may be a sign that the dependencies are not set up correctly for the case when Cotire is enabled, leading to an undefined reference (something that needs to be built is not being built).